runtime/src/accounts_db.rs

```diff
@@ -1515,7 +1528,7 @@ impl AccountsDb {
 
         let mut collect_delta_keys = Measure::start("key_create");
         let max_slot = max_clean_root.unwrap_or_else(|| self.accounts_index.max_root());
-        let (delta_keys, _max_slot) = self.collect_uncleaned_pubkeys_to_slot(max_slot);
```

Comment:
For more discussion about removing the max_slot return value, refer to https://github.com/solana-labs/solana/pull/16786#pullrequestreview-643591378